scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/soic.yaml

```diff
@@ -350,6 +349,27 @@ SOIC-8-1EP_3.9x4.9mm_P1.27mm_EP2.514x3.2mm:
     # bottom_pad_size:
     paste_avoid_via: False
 
+SOIC-8_5.275x5.275mm_P1.27mm:
+  size_source: 'http://ww1.microchip.com/downloads/en/DeviceDoc/20005045C.pdf'
```

Comment:
add #page=23 to point directly to the correct page

Reply:
Done